src/App.tsx

```diff
@@ -1,5 +1,25 @@
+import { ThemeProvider } from '@emotion/react';
+import { createTheme } from '@mui/material/styles';
+
 import { Router } from './Router';
 
+const theme = createTheme({
```

Comment:
That should be moved to separate file like **config/theme.ts**

Reply:
ok, i will do it